src/environments/environment.prod.ts

```diff
@@ -1,4 +1,6 @@
 export const environment = {
   production: true,
-  LOG_LEVEL: 'warn'
+  LOG_LEVEL: 'warn',
+  AUTH_URL: 'https://accounts.google.com/o/oauth2/v2/auth',
+  OAUTH_CLIENT_ID: '603712460284-9c20vfg98ra1cecdhu2r84d4ij6erpcr.apps.googleusercontent.com'
```

Comment:
Is this CLIENT_ID permanent?

Also - can we add this in README.md section on OAuth that such client is needed and how do we register to get it?